core/domain/stats_jobs_continuous.py

```diff
@@ -363,3 +365,90 @@ def reduce(key, stringified_values):
         stats_models.ExplorationAnnotationsModel.create(
             exp_id, str(version), num_starts, num_completions,
             state_hit_counts)
+
+
+class InteractionAnswerSummariesMRJobManager(
+        jobs.BaseMapReduceJobManagerForContinuousComputations):
+    """
+    Job to calculate interaction view statistics, e.g. most frequent
+    answers of multiple-choice interactions. Iterate over StateAnswer
+    objects and create StateAnswersCalcOutput objects.
+    """
+    @classmethod
+    def _get_continuous_computation_class(cls):
+        return InteractionAnswerSummariesAggregator
+
+    @classmethod
+    def entity_classes_to_map_over(cls):
+        return [stats_models.StateAnswersModel]
+
+    @staticmethod
+    def map(item):
+        from core.domain import calculation_registry
+        from core.domain import interaction_registry
+
+        if InteractionAnswerSummariesMRJobManager._entity_created_before_job_queued(
+                item):
+
+            # All visualizations desired for the interaction.
+            visualizations = interaction_registry.Registry.get_interaction_by_id(
+                item.interaction_id).answer_visualizations
+
+            # Get all desired calculations for the current state interaction id.
+            calc_ids = list(set(viz.calculation_id for viz in visualizations))
+            calculations = [
+                calculation_registry.Registry.get_calculation_by_id(calc_id)
+                for calc_id in calc_ids]
+
+            # Perform each calculation, and store the output.
+            for calc in calculations:
+                calc_output = calc.calculate_from_state_answers_entity(item)
+                calc_output.save()
+
+    @staticmethod
+    def reduce(id, state_answers_model):
+        pass
+
+
+class InteractionAnswerSummariesRealtimeModel(
+        jobs.BaseRealtimeDatastoreClassForContinuousComputations):
+    num_starts = ndb.IntegerProperty(default=0)
```

Comment:
I don't believe so. I replaced the properties with a TODO to implement a real-time layer for the interaction answer summaries aggregator job.